content/GPR/GPR_fundamental_principles.rst

```diff
@@ -0,0 +1,402 @@
+
+
+When a radiowave reaches an interface, some of it is reflected and some of it is transmitted across the interface.
+This results in both a reflected and a transmitted wave.
+
+The amplitude of the reflected wave proportional to that of the incident wave is defined by the reflection coefficient (:math:`R`).
+For radiowaves, the reflection coefficient can be expressed as a function of the dielectric permittivities on each side of the interface.
+Assuming the radiowave arrives at an angle perpendicular to the interface, the reflection coefficient is given by:
+
+.. math::
+	R = \frac{\textrm{Reflected Amplitude}}{\textrm{Incident Amplitude}} = \frac{\sqrt{\varepsilon_1} - \sqrt{\varepsilon_2}}{\sqrt{\varepsilon_1} + \sqrt{\varepsilon_2}}
+
+
+where :math:`\varepsilon_1` is the dielectric permittivity of the medium carrying the incident and reflected waves.
+
+The reflection coefficient can be either positive or negative and has values between :math:`-1 < R < 1`.
+The magnitude of :math:`R` determines how much of the incident wave is reflected.
+It should be noted that:
+
+	- If :math:`\varepsilon_1` and :math:`\varepsilon_2` are similar, most of the incident wave is transmitted through the interface.
+	- If one of the dielectric permittivities across the interface is much smaller than the other, most of the incident wave is reflected. This can be a problem if you at attempting to gain information about structures below this interface.
+
+
+The sign of the reflection coefficient determines whether the reflected wave experiences a reverse in polarity.
+As a result, we can use the polarity of reflected radiowaves to determine whether :math:`\varepsilon_1` is greater than or less than :math:`\varepsilon_2`.
+This can be summarized as follows:
+
+	- If the returning signal (reflected wave) shows a reverse in polarity, :math:`R<0` and thus :math:`\varepsilon_1 < \varepsilon_2`
+	- If the returning signal (reflected wave) does not show a reverse in polarity, :math:`R>0` and thus :math:`\varepsilon_1 > \varepsilon_2`
+
+
+Reflection at Conductive Object Boundaries
+==========================================
+
+Many compact objects relevant to GPR applications (buried pipes, landmines, etc...) are highly conductive.
+For highly conductive objects, the reflection and transmission of radiowaves cannot be characterized by the dielectric permittivities.
+Instead, we must consider the electrical conductivity of the object.
+
+From the general equation for propagation velocity, we see that as :math:`\sigma \rightarrow \infty`, the propagation velocity goes to zero.
+Thus radiowaves cannot propagate through extremely conductive objects.
+Because of this, when radiowaves reach the interface between the Earth and a highly conductive object, the wave is completely reflected.
+This occurs regardless of the incident angle.
+
+
+Refraction of Radiowaves
```

Comment:
could you please add a reference in this section to the corresponding section in the seismic basic principles?